packages/web/src/components/list/SingleDataList.js

```diff
@@ -398,15 +402,27 @@ class SingleDataList extends Component {
 									id={`${this.props.componentId}-${selectAllLabel}`}
 									value={selectAllLabel}
 									tabIndex={isAllChecked ? '-1' : '0'}
-									onChange={this.handleClick}
+									onClick={this.handleClick}
```

Comment:
@SavvyShah What is the reason of this change?

Reply:
Because it was used by other [list items rendered through `data` property](https://github.com/appbaseio/reactivesearch/blob/33fa468f3c15f1f61431373cf7c53312a28d7304/packages/web/src/components/list/SingleDataList.js#L446), so to create consistency.